routes/api/report.js

```diff
@@ -0,0 +1,88 @@
+var express = require('express');
+var router = express.Router();
+var utils = require('../../utils.js');
+var logger = require('log4js').getLogger();
+logger.level = 'debug';
+
+module.exports = function(connection) {
+
+    router.get('/report/module', function (req, res, next) {
+
+        connection.query('select data from atlas', function (error, rows, field) {
+
+            if(error) {
+                logger.error(error);
+            } else {
+                var resp = [];
+                var idxs = {};
+
+                rows.forEach(function(row) {
+                    var data = row.data;
+                    if(data && utils.isJson(data)) {
+                        data = JSON.parse(data);
+                        if(data.modules) {
+                            data.modules.forEach(function(module) {
+                                if(module.active !== "true") return;
+                                if(!idxs[module.name]) {
+                                    idxs[module.name] = resp.length;
+                                    resp.push({ id: module.name.toLowerCase(), name: module.name, versions: {} });
```

Comment:
I don't understand this. Can we use `module.id` (the module's actual id) instead of creating a fake id from the name? I believe the atlas module sends module ids. Module id is, by convention, used as a java package name when creating a module, so follows the rules of a java token – i.e., will not contain spaces.